### PR TITLE
Fix crash on string in multi-field uniques in datamodel

### PR DESCRIPTION
**Pull request by tomhoule**
closes https://github.com/prisma/prisma/issues/5985

### CODE REVIEW

File: libs/datamodel/core/src/transform/attributes/unique_and_index.rs [hunk 1]
```diff
@@ -146,8 +146,8 @@ trait IndexAttributeBase<T>: AttributeValidator<T> {
             .default_arg("fields")?
             .as_array()
             .iter()
-            .map(|f| f.as_constant_literal().unwrap())
```
Comment (do4gr) [inline libs/datamodel/core/src/transform/attributes/unique_and_index.rs:149]: Can you do the same fix for @@id as well? I checked there we also unwrap

Comment (do4gr) [reply libs/datamodel/core/src/transform/attributes/unique_and_index.rs:149]: Apart from this 👍

Comment (tomhoule) [reply libs/datamodel/core/src/transform/attributes/unique_and_index.rs:149]: Good catch! Will add as an extra commit in this PR.

Comment (do4gr) [reply libs/datamodel/core/src/transform/attributes/unique_and_index.rs:149]: Great!

**Pull request merged**